src/main/java/mil/dds/anet/search/AbstractLocationSearcher.java

```diff
@@ -53,6 +54,15 @@ protected void buildQuery(LocationSearchQuery query) {
       addLocationUuidQuery(query);
     }
 
+    final BoundingBox bbox = query.getBoundingBox();
+    if (bbox != null) {
+      qb.addWhereClause("(lng BETWEEN :minLng AND :maxLng AND lat BETWEEN :minLat AND :maxLat)");
```

Comment:
This doesn't work when the bounds wrap the international dateline (180° E / 180° W).